src/util/BaseLoginRouter.js

```diff
@@ -229,7 +233,7 @@ function (Okta, Backbone, BrowserFeatures, xdomain, RefreshAuthStateController,
           Beacon = null;
         }
         this.header.setBeacon(Beacon, controllerOptions);
-
+        this.header.setIsSecurityBeaconClass(isSecurityBeacon(Beacon));
```

Comment:
We shouldn't add page specific logic to the BaseLoginRouter (or generic classes in general).

Related to the other comment I left below:
- The view that knows its a security beacon is SecurityBeacon.
- It should not update dom elements outside of itself, so it will need to a) pass a message b) set appState to communicate this information to a higher level view.
- The higher level view listens for this message/change in state and then sets the class.